src/Cdn/Cdn/help/Az.Cdn.md

```diff
@@ -62,6 +62,18 @@ Gets the resource usage for a susbscription.
 ### [New-AzCdnCustomDomain](New-AzCdnCustomDomain.md)
 Creates a custom domain for a CDN endpoint.
 
+### [New-AzCdnDeliveryPolicy](New-AzCdnDeliveryPolicy.md)
+{{ Fill in the Synopsis }}
```

Comment:
@hytao please fill in these `{{ Fill in the Synopsis }}` templates with a brief overview of the cmdlets' purpose